client/src/main/webjar/toolbar/toolbar.html

```diff
@@ -18,6 +18,18 @@
              src="/images/logo-64px.png"
              alt="Healthy Mothers, Healthy Babies Coalition of Georgia">
 
+        <span flex></span>
+
+        <md-button href="http://www.resourcehouse.com/HMHB/"
+                   target="_blank"
+                   rel="noopener noreferrer">
```

Comment:
This line protects against phishing window.opener vulnerability.

FYI:
https://dev.to/phishing
https://dev.to/ben/the-targetblank-vulnerability-by-example